tests/cpp_methods/cppfilter_test.cc

```diff
@@ -180,8 +176,6 @@ filter_basic2::getOutputDim (GstTensorsInfo *info)
   info->info[0].dimension[1] = 16;
   info->info[0].dimension[2] = 16;
   info->info[0].dimension[3] = 2;
-  for (int i = 4; i < NNS_TENSOR_RANK_LIMIT; i++)
-    info->info[0].dimension[i] = 1;
```

Comment:
You cannot sure if info is properly initialized.

Reply:
I added init code to get in/out dims. plz review the changes.